apps/frontend-pwa/src/components/practiceQuiz/InstanceHeader.tsx

```diff
@@ -20,12 +28,43 @@ interface InstanceHeaderProps {
 
 function InstanceHeader({
   instanceId,
+  elementId,
   name,
   withParticipant,
   correctness,
   className,
 }: InstanceHeaderProps) {
+  const [rateElement] = useMutation(RateElementDocument)
   const [modalOpen, setModalOpen] = useState(false)
+  const [vote, setVote] = useState(0) // TODO: optionally fetch last rating from DB
```

Comment:
Should we fetch the previous submitted feedback here? The added index should allow to fetch the corresponding entry relatively fast with the participantId and elementInstanceId being known?

Reply:
if we do that, we should do it in a scalable way and not send x frontend requests in a stack with x items. instead fetch it once for the entire stack by sending all ids of the elements in the stack. or even already in the request where we get the questions for the stack from DB (we can join and inside the join add a where clause with the two elements of your index, instance id and participant id?) 

-> if we do it like that, it might be relatively efficient. but not a large priority to have it I think, as it is probably not used so often anyways

Reply:
--> will be implemented on separate PR